posts/templatetags/parser.py

```diff
@@ -0,0 +1,24 @@
+"""@package docstring
+Extra template functions
+
+@author 7Pros
+@copyright
+"""
+from django import template
+from django.core.urlresolvers import reverse
+from posts.views import PostsListView
+from django.template.defaultfilters import stringfilter
+import re
+
+register = template.Library()
+
+@register.filter
+@stringfilter
+def parse(content):
```

Comment:
The name of this filter is too ambiguous. `parse` could mean anything.
Something like `parse-post-content` or `insert-hashtag-links` would make it clearer what this filter does.